### PR TITLE
root: refactored msi packager to match the latest implementation

### PR DESCRIPTION
**Pull request by hollowaykeanho**
The MSI packager was intentionally left out due to its complications and issue listed in:
       https://github.com/wixtoolset/issues/issues/7896
Now that the overall package job and release job are stabilized, we can proceed to refactor MSI packager now. Hence, let's do this.

This patch refactors MSI packager to match the latest implementation in root repository.

### CODE REVIEW

File: src/.ci/_x-package-msi_windows-any.ps1 [hunk 1]
```diff
@@ -0,0 +1,12 @@
+		switch ($__i18n) {
```
Comment (corygalyna) [inline src/.ci/_x-package-msi_windows-any.ps1]: What is the function of this file? If it's a leftover, please remove it.

**Comment by hollowaykeanho** [comment]
development artifact removed.

**Review by corygalyna — APPROVED** [review]
Nice. All cleared. Approved.

File: src/.ci/_x-package-msi_windows-any.ps1 (hunk 1 above)
Comment (hollowaykeanho) [reply src/.ci/_x-package-msi_windows-any.ps1]: done.

**Pull request merged**